crates/bevy_app/src/app.rs

```diff
@@ -1081,28 +1089,6 @@ impl App {
         self
     }
 
-    /// Adds an [`App`] as a child of the current one.
-    ///
-    /// The provided function `extract` is normally called by the [`update`](Self::update) method.
-    /// After extract is called, the [`Schedule`] of the sub app is run. The [`World`]
-    /// parameter represents the main app world, while the [`App`] parameter is just a mutable
-    /// reference to the `SubApp` itself.
```

Comment:
The description of the extract function should probably be moved somewhere else. Maybe to the `SubApp` docs?

Reply:
Done.